ensdomains/ens-app-v3
# update useProfile to have option to get abi record

**Change List**
- useABI
  - Moved useABI from useNameDetails to useProfile
  - Added option to get from specific resolver
- Removed type DetailedProfile and moved abi typing to type Profile
- Increased "expect" timeout to 15000, this will cover the loading time of Skeleton component
- Updated the following hooks to follow coding standards
  - useResolverType
  - useABI
  - useNameDetail
  - usePrimaryProfile
  - useProfile
- Added ABI touple function so it can be listed as a change in updateRecords transaction info items
- Updated profile comparison functions to include abi

**How to test**
- Add/Update/Delete abi in profile and advanced editor and check that it shows up in list of changes
- Migrate profile including abi with outdated resolver where latest resolver has and does not have a profile. (Test "migrate profile" and "migrate profile with reset")
  - You will want to test where the latest resolver has: 
    - no records (test "migrate-profile")
    - the same abi record (test "isABIEqual" func")
    - different abi record, choosing current profile (test "migrate-profile-with-reset")
